docs/team/ovreader.md

```diff
@@ -0,0 +1,51 @@
+# Ang Jia Le Marcus - Project Portfolio Page
+
+## Property Rental Manager - Overview
+Property Rental Manager (PRM) is a desktop application which operates via Command Line Interface (CLI) that helps property agents manage, filter and monitor single owner rental units for appropriate tenants. This application targets specifically property agents who are working in Singapore and dealing with local properties. Thus, some features are tailored to Singapore context. 
+
+Given below are my contributions to the project.
+
+- **New feature**: Added the ability to add new property, along with various property detail validations.
```

Comment:
Maybe this sentence could be better phrase? The portion on "Added the ability to add" sound slightly repetitive